clients/admin-ui/src/features/common/custom-fields/CreateCustomFields.tsx

```diff
@@ -158,109 +140,25 @@ const CreateCustomFields = forwardRef(
-                    field,
-                    meta,
-                  }: {
-                    field: FieldInputProps<string>;
-                    meta: FieldMetaProps<string>;
-                  }) => (
-                    <FormControl
-                      display="flex"
-                      isRequired
-                      isInvalid={!!(meta.error && meta.touched)}
-                    >
-                      <FormLabel
-                        color="gray.600"
-                        fontSize="14px"
-                        fontWeight="semibold"
-                        minWidth="150px"
-                      >
-                        Select custom list
-                      </FormLabel>
-                      <VStack align="flex-start" flexGrow={1}>
-                        <SelectDropdown
-                          enableSorting={false}
-                          hasClear={false}
-                          label="Select..."
-                          list={list}
-                          menuButtonProps={{
-                            textAlign: "left",
-                            width: "100%",
-                          }}
-                          onChange={(value: string | undefined) => {
-                            props.setFieldValue("allow_list_id", value);
-                          }}
-                          selectedValue={props.values.allow_list_id || ""}
-                        />
-
-                        <FormErrorMessage>{meta.error}</FormErrorMessage>
-                      </VStack>
-                    </FormControl>
-                  )}
-                </Field>
-                <Field name="resource_type">
-                  {({ field }: { field: FieldInputProps<string> }) => (
-                    <FormControl display="flex">
-                      <FormLabel
-                        color="gray.600"
-                        fontSize="14px"
-                        fontWeight="semibold"
-                        minWidth="150px"
-                      >
-                        Resource type
-                      </FormLabel>
-                      <VStack align="flex-start" flexGrow={1}>
-                        <SelectDropdown
-                          disabled
-                          enableSorting={false}
-                          hasClear={false}
-                          label=""
-                          list={RESOURCE_TYPE_MAP}
-                          menuButtonProps={{
-                            textAlign: "left",
-                            width: "100%",
-                          }}
-                          onChange={(value: string | undefined) => {
-                            props.setFieldValue("resource_type", value);
-                          }}
-                          selectedValue={props.values.resource_type}
-                        />
-                      </VStack>
-                    </FormControl>
-                  )}
-                </Field>
```

Comment:
[Post-review]: The `SelectDropdown` was an old component. `CustomSelect` has more functionality.